pom.xml

```diff
@@ -70,10 +70,16 @@ THE SOFTWARE.
 
   <repositories>
     <repository>
-      <id>mavenTMateSoftReleasesRepository</id>
-      <url>http://maven.tmatesoft.com/content/repositories/releases/</url>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
     </repository>
   </repositories>
+  <pluginRepositories>
+    <pluginRepository>
+      <id>repo.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/public/</url>
+    </pluginRepository>
+  </pluginRepositories>
```

Comment:
@jglick But, this configuration is defined (inherited) in parent [POM](https://github.com/jenkinsci/plugin-pom/blob/master/pom.xml#L711), right?